services/checks/registry.go

```diff
@@ -67,18 +66,18 @@ func NewRegistry() *Registry {
 	}
 }
 
-// CreateAlert creates alert from given AlertParams and adds or replaces alert with given ID in registry.
+// createAlert creates alert from given AlertParams and adds or replaces alert with given ID in registry.
 // If that ID wasn't present before, alert is added in the pending state. It we be transitioned to the firing
 // state after delayFor interval. This is similar to `for` field of Prometheus alerting rule:
 // https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
-func (r *Registry) CreateAlert(id string, labels, annotations map[string]string, delayFor time.Duration) {
+func (r *registry) createAlert(id string, labels, annotations map[string]string, delayFor time.Duration) {
```

Comment:
I wonder, do we need delay for STT alerts at all? 🤔

Reply:
Yes, let's get rid of delayFor completely – it was over-engineered, and it is clear now that our future "dynamic alerting" feature will work differently